website/docs/best-practices/how-we-mesh/mesh-4-faqs.md

```diff
@@ -0,0 +1,297 @@
+
+</detailsToggle>
+
+<detailsToggle alt_header="Is it possible to request access permissions from other teams?">
+
+There is not currently! But this is something we may evaluate for the future.
+
+</detailsToggle>
+
+<detailsToggle alt_header="Can projects be hidden for confidentiality?">
+
+The "multi-project" view in dbt Cloud Explorer (and the underlying Discovery API) will include any project that has defined at least one "public" model, including the list of public models. (Read more about [model access modifiers](/docs/collaborate/govern/model-access#access-modifiers).) If a user has additional permissions on that project (managed via dbt Cloud RBAC), including read-only permissions, they can explore further into that project and see details about all models (private, protected, and protected).
+
+</detailsToggle>
+
+<detailsToggle alt_header="As a central data team member, can I still maintain visibility on the entire organizational DAG?">
+
+Yes! As long as a user has permissions (at least read-only access) on all projects in a dbt Cloud account, they can navigate across the entirety of the organization’s DAG in dbt Explorer, and see models at all levels of detail.
+
+</detailsToggle>
+
+<detailsToggle alt_header="TO DO -- How do model-level access, dbt Cloud's role-based access, and data platform access interact?">
+
+OUTSTANDING COPY
+
+</detailsToggle>
+
+<detailsToggle alt_header="TO DO -- How can I limit my developers from accessing production data when referencing from other projects?">
+
+My production environments contain sensitive data. How can I limit my developers from accessing production data when referencing from other projects?
+
+</detailsToggle>
+
+## Compatibility with other features
+
+<detailsToggle alt_header="How does the dbt Semantic Layer relate to and work with dbt Mesh?">
+
+The [dbt Semantic Layer](/docs/use-dbt-semantic-layer/dbt-sl) and dbt Mesh are complementary mechanisms enabled by dbt Cloud that work together to enhance the management, usability, and governance of data in large-scale data environments.
+
+The Semantic Layer in dbt Cloud allows teams to centrally define business metrics and dimensions. It ensures consistent and reliable metric definitions across various analytics tools and platforms.
+
+dbt Mesh enables organizations to split their data architecture into multiple domain-specific projects, while retaining the ability to reference “public” models across projects. It is also possible to reference a “public” model from another project for the purpose of defining semantic models and metrics. In this way, your organization can have multiple dbt projects feed into a unified semantic layer, ensuring that metrics and dimensions are consistently defined and understood across these different domains.
+
+</detailsToggle>
+
+<detailsToggle alt_header="TO DO -- Can dbt Mesh help control permissions in the Semantic Layer? How does it work with semantic tokens?">
```

Comment:
needs answer filled in: [direct link to notion question](https://www.notion.so/WIP-dbt-Mesh-Customer-FAQ-bdadabc9917a4be697db53142c6ff979?pvs=4#0629231db0d34ce28a6ecbed403f3c13)